ci/container/build-ic.sh

```diff
@@ -172,7 +172,10 @@ for artifact in $(bazel cquery "${BAZEL_COMMON_ARGS[@]}" --output=files "$query"
     esac
 
     mkdir -p "$target_dir"
-    cp "$artifact" "$target_dir"
+
+    # We use -L so that find dereferences symlinks (artifacts are not
+    # necessarily duplicated in the build)
+    find -L "$artifact" -type f -exec cp {} "$target_dir" \;
```

Comment:
Why not:
```suggestion
    cp -L "$artifact" "$target_dir"
```

Reply:
the artifacts are actually nested inside `$artifact` undefined `$artifact/<prefix>`. The `$artifact` is the root directry which we need to explore first. Does this make sense?